openedx/core/djangoapps/theming/models.py

```diff
@@ -0,0 +1,39 @@
+"""
+Django models supporting the Comprehensive Theming subsystem
+"""
+from django.db import models
+from django.conf import settings
+from django.contrib.sites.models import Site
+
+
+class SiteTheme(models.Model):
```

Comment:
I think combined with the discussion below re: theme stacking, we want a separate model vs. a simple field on a SiteConfiguration model, although it would be possible for the SiteConfiguration model field to contain an array of themes.  I guess it depends on how we see themes evolving down the road -- ie, will there be more metadata around individual themes that we want to be able to control via the admin.